openapi/code/service.go

```diff
@@ -202,9 +208,14 @@ func (svc *Service) newMethodEntity(op *openapi.Operation) (*Entity, openapi.Mim
 	return res, mimeType, bodyField
 }
 
-func (svc *Service) addParams(request *Entity, op *openapi.Operation, params []openapi.Parameter) {
+func (svc *Service) skipHeader(v openapi.Parameter, includeHeaders bool) bool {
+	hiddenHeader := HIDDEN_HEADERS[v.Name]
+	return v.In == "header" && (!includeHeaders || hiddenHeader)
+}
+
+func (svc *Service) addParams(request *Entity, op *openapi.Operation, params []openapi.Parameter, includeHeaders bool) {
```

Comment:
Why do we need `includeHeaders`? Is `includeHeaders` only `false` on the request pathway? I imagine the `skipHeader` method is sufficient? What other headers are referenced in the spec?